subworkflows/local/gtdbtk.nf

```diff
@@ -14,40 +14,25 @@ workflow GTDBTK {
     gtdb_mash         // channel: path
 
     main:
-    // Filter bins: classify only medium & high quality MAGs
-    ch_bin_metrics = Channel.empty()
-    if ( params.binqc_tool == 'busco' ){
-        // Collect completeness and contamination metrics from busco summary
-        ch_bin_metrics = bin_qc_summary
-            .splitCsv(header: true, sep: '\t')
-            .map { row ->
-                        def completeness  = -1
-                        def contamination = -1
-                        def missing, duplicated
```

Comment:
This deletion means that the pipeline is no longer backwards compatible with older versions of the database, correct? This is important to list in te changelog

Reply:
Yes, but it's BUSCO itself that doesn't support older versions of downloaded databases, as it now expects the new odb12 lineages (which is why you encountered that error).

Maybe it's better to add a notice in the release instead of the changelog?